apps/storybook-e2e/src/e2e/components/steps/vl-steps.stories.cy.ts

```diff
@@ -84,14 +84,13 @@ describe('story vl-steps', () => {
             .should('have.class', 'vl-step--error');
     });
 
-    // TODO kspeltin: waarom faalt deze test ?
-    it.skip('should contain a clickable button ', () => {
+    it('should contain a clickable button ', () => {
         // Check closed state
         cy.visit(`${stepsWithAccordionsUrl}`);
         cy.getDataCy('steps-with-accordions')
```

Comment:
dit is technisch niet verkeerd en in lijn met hetgene dat er al stond, maar we proberen typisch die `getDataCy()` weg te werken als we de test aanpassen door gewoon `get()`

Reply:
Ja, ik had gewoon uit interesse de skip eens weggenomen en een kleine change gedaan en het werd groen, had er voor de rest niet echt naar gekeken.
Heb het aangepast